Z/ZXing_CPP/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ZXing_CPP"
+version = v"2.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/zxing-cpp/zxing-cpp.git", "d6068bcebeb8fd9f0d35a99b00d202be86a14dbe")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd zxing-cpp/
+git submodule update --init
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_CXX_STANDARD=20 -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DZXING_READERS=ON -DZXING_WRITERS=NEW -DZXING_USE_BUNDLED_ZINT=ON -DZXING_C_API=ON -DZXING_EXPERIMENTAL_API=ON
+cmake --build build --parallel ${nproc} --config Release
```

Comment:
You're already asking for the Release build type, right?
```suggestion
cmake --build build --parallel ${nproc}
```

Reply:
Yes